rest/blip_sync_messages.go

```diff
@@ -40,7 +40,7 @@ func (s *subChangesParams) since() (db.SequenceID, error) {
 
 	if sinceStr, found := s.rq.Properties["since"]; found {
 		var err error
-		if sinceSequenceId, err = s.sequenceIDParser(sinceStr); err != nil {
+		if sinceSequenceId, err = s.sequenceIDParser(base.ConvertJSONString(sinceStr)); err != nil {
```

Comment:
In the numeric sequence case, what's sinceStr going to be?  `'123'`, or `'"123"'` ?

Does ConvertJSONString handle both?

Reply:
Yeah `ConvertJSONString()` handles both -- I added a unit test and updated the comments accordingly